workshop/Lab0/README.md

```diff
@@ -0,0 +1,112 @@
+```
+You should see the repository listed in the output of `helm search service-catalog`.
+
+2. Install Service Catalog by running 
+```
+$ helm install svc-cat/catalog --name catalog --namespace catalog
+```
+
+You should be able to see the helm tiller and service catalog pods like so:
+```console
+$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                                                  READY     STATUS    RESTARTS   AGE
+catalog       catalog-catalog-apiserver-6bc6bb895d-f2xhp            2/2       Running   0          8m
+catalog       catalog-catalog-controller-manager-5c7475f7d4-fwlfg   1/1       Running   1          8m
+kube-system   kube-addon-manager-minikube                           1/1       Running   0          8m
+kube-system   kube-dns-54cccfbdf8-vpvmn                             3/3       Running   0          8m
+kube-system   kubernetes-dashboard-77d8b98585-sw5gc                 1/1       Running   0          8m
+kube-system   storage-provisioner                                   1/1       Running   0          8m
+kube-system   tiller-deploy-865dd6c794-88njn                        1/1       Running   0          9m
+```
+
+# Install svcat
+
+svcat is a command line tool for interacting with Service Catalog. While Service Catalog
+can be intereacted with solely with kubectl, svcat provides a nicer UX that is customized
+to interact with Service Catalog specific resource types. It can be run as a kubectl plugin,
+or as a standalone binary. Download the version that corresponds to your operating system:
+
+* [Linux](https://download.svcat.sh/cli/latest/linux/amd64/svcat)
+* [OS X](https://download.svcat.sh/cli/latest/darwin/amd64/svcat)
+* [Windows](https://download.svcat.sh/cli/latest/windows/amd64/svcat.exe)
+
+Make the binary executable:
+
+* (Linux/OS X) `chmod +x svcat`
+* (Windows) Make sure the filename ends in .exe
+
+Move the file somewhere on your PATH:
+
+* (Linux/OS X) `mv svcat /usr/local/bin/`
+* (Windows) `mkdir -f ~\bin && $env:PATH += ";${pwd}\bin"`
+
+Ensure that you can see svcat and it can access your Kubernetes deployment:
+```console
+$ svcat get classes
+  NAME   DESCRIPTION   UUID  
++------+-------------+------+
+```
+
+# Download the Workshop Source Code
+
+The 'svccat' repository contains the step by step instructions and some yaml manifests
+needed to run the workshop.
+
+```
+$ git clone https://github.com/IBM/svccat.git
+```
+
+# Create a Private Volume
+
+The service broker we will be deploying as part of this workshop will deploy resources into your Kubernetes cluster. In order
+for these services to have a place to store data, we need to create a Persistant Volume for them to mount. Find the `pv.yaml`
+file in the Lab0 directory of this repository, and then run 
+```
+kubectl create -f pv.yaml`
+```
+This step is only needed to support the specific service broker we'll be using on IBM Cloud IKS. It is not a normal step of the
```

Comment:
just to be clear... its actually the "service" we're using that makes use of the volume, not the broker itself, right?
If so, how about:
This step is only needed to support the specific service (redis) that we'll be deploying using the service broker on the IBM Cloud IKS.